zio-http/shared/src/main/scala/zio/http/RoutePattern.scala

```diff
@@ -164,6 +164,18 @@ object RoutePattern {
    */
   def fromMethod(method: Method): RoutePattern[Unit] = RoutePattern(method, PathCodec.empty)
 
+  /**
+   * Creates a RoutePattern that matches any of the specified paths.
+   */
+  def anyOf(paths: String*): PathCodec[Unit] = {
+    if (paths.isEmpty) {
+      PathCodec.empty
```

Comment:
This is unnecessary, no? should be handled by the `getOrElse`

Reply:
Yes, just added to see the test fail causing of the empty path to check beforehand, will remove it 😅.

@

Reply:
@varshith257 The tests that are failing are the ones that you added as part of the PR. It means that the implementation doesn't work